### PR TITLE
add Travis-CI support

### PR DESCRIPTION
**Pull request by raveit65**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    - make
+    - mate-common
+    - pkg-config
+
+variables:
+  - CFLAGS="-Wall -Werror=format-security"
+
+before_scripts:
+  - cd ${START_DIR}
+  - if [ ! -f caja-1.21.4.tar.xz ];then
```
Comment (yetist) [inline .travis.yml]: Not sure if we have to compile caja 1.21, but it looks like there is no problem.

Comment (raveit65) [reply .travis.yml]: Opps, 1.20.2 might be better for 1.20 branch.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    - libnotify-dev # for caja build
+    - libstartup-notification0-dev
+    - make
+    - mate-common
+    - pkg-config
+
+variables:
+  - CFLAGS="-Wall -Werror=format-security"
+
+before_scripts:
```
Comment (yetist) [inline .travis.yml]: Maybe we can remove this part? caja-devel has installed use package manage already.

Comment (raveit65) [reply .travis.yml]: Should be sufficient. You mean the whole part of `before_scripts`?

Comment (yetist) [reply .travis.yml]: Yes, remove the whole `before_scripts` part, include the section name: `before_scripts:`

Comment (raveit65) [reply .travis.yml]: ok, this works fine.
Any other concerns?
If not just merge it please.

**Pull request merged**